documentation/latest/operations/prebuilt.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: documentation
+title: Prebuilt WorkFlow tasks
+---
+
+Tasks such as REST requests and Kubernetes operations are common to
+many workflows and applications.
+We provide a collection of ready to use workflow tasks. A.k.a prebuilt tasks.
+Example for usage can be found in the
+[Parodos examples module](https://github.com/parodos-dev/parodos/tree/f2bb45ff0813b31dc76a8b80176efb567952c1eb/workflow-examples/src/test/java/com/redhat/parodos/examples).
+
+## Create Azure VM - AzureCreateVirtualMachineTask
+
+A task for creating a VM in Azure. It allows specifying a username and SSH key
+for accessing the VM. The VM is created with UBUNTU image.
+An example is available
+[here](https://github.com/parodos-dev/parodos/tree/f13756c822d481c13f791c473097efc695fdbbd6/workflow-examples/src/main/java/com/redhat/parodos/examples/azure)
+
+## Deploy OpenShift application - DeployApplicationTask
+
+A task for applying manifests to an OpenShift cluster.
+User provides a folder with manifests (YAML files) and the task applies them
+one by one. The user supplies a kubeconfig string for authentication and
+connection details.
+
+## Git tasks
+
+We have a collection of Git operation tasks:
+
+- GitPushTask
+- GitArchiveTask
+- GitCloneTask
+- GitCommitTask
+- GitPushTask
+
+User can, optionally, provide Git credentials.
+
+## JDBC CRUD operation - JdbcWorkFlowTask
+
+The task uses JDBC for interacting with a database.
+The operations that the task executes are divided into three types.
+The type depends on the commands/statement that user provides.
+
+- query. E.g. SQL select
+- update. E.g. SQL delete, insert, update
+- execute. E.g. SQL drop table, create table
+
+The user provides the JDBC connection URL. In the URL various
+parameters can be included.
+Username, password, TLS etc. Each JDBC driver has its own specific parameters.
```

Comment:
Should `Username, password, TLS etc.` be part of the previous sentence?

Reply:
there is a limit of 80 chars in line. so I have to split the lines. when displayed to user it looks as a single line.
![image](https://github.com/parodos-dev/parodos-dev.github.io/assets/69590123/e31aa28b-45ac-445a-b146-202d8312984a)